RadixWallet/Features/EditPersonaFeature/EditPersona/EditPersona.swift

```diff
@@ -57,16 +57,16 @@ extension EditPersona {
 public struct EditPersona: Sendable, FeatureReducer {
```

Comment:
Didn’t find a better naming that will suit both Create and Edit flows. Open to suggestions.